src/routes/cohort.js

```diff
@@ -8,5 +8,6 @@ import {
 const router = Router()
 
 router.post('/', validateAuthentication, validateTeacherRole, create)
+router.get('/:id', validateAuthentication, getById)
```

Comment:
change to `/:id/users` since we are getting the cohort's users and not the whole cohort data